Dockerfile

```diff
@@ -3,7 +3,7 @@
 #
 # See CONTRIBUTING.md
 #
-FROM mcr.microsoft.com/dotnet/sdk:8.0
+FROM mcr.microsoft.com/dotnet/sdk:9.0
```

Comment:
Berp does not work on .net 9.

But I haven't figured out why Renovate makes this PR immortal.